static/js/Misc.jsx

```diff
@@ -3369,9 +3375,15 @@ const LangSelectInterface = ({callback, defaultVal, closeInterface}) => {
       }
     >
       <div className="langHeader"><InterfaceText>Source Language</InterfaceText></div>
-      <LangRadioButton buttonTitle="Source Language" lang={lang} buttonId="source" handleLangChange={handleLangChange}/>
-      <LangRadioButton buttonTitle="Translation" lang={lang} buttonId="translation" handleLangChange={handleLangChange}/>
-      <LangRadioButton buttonTitle="Source with Translation" lang={lang} buttonId="sourcewtrans" handleLangChange={handleLangChange}/>
+       {buttonData.map((button, index) => (
+        <LangRadioButton
+          key={index}
```

Comment:
don't use index as key! this is an anti-pattern (see here https://reactpatterns.js.org/docs/indexes-as-a-key-is-an-anti-pattern/). Instead use buttonId